website/docs/docs/contributing/adapter-development/1-what-are-adapters.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "What are adapters? Why do we need them?"
+id: "1-what-are-adapters"
+---
+
+Here is a quick intro as to why adapters need to exist and how they are currently constructed. For any questions you may have, don't hesitate to ask in the [#adapter-ecosystem](https://getdbt.slack.com/archives/C030A0UF5LM) Slack channel. The community is very helpful and likely has experienced a similar issue as you.
```

Comment:
Let's lead with a two or three-sentence summary of why adapters need to exist and close this section with the Slack channel.  We can move the description of the doc to the headers (if need be) to enhance SEO.

Reply:
are you saying "lead with" as in you want me to write more sentences in place of what is currently there?

Reply:
Could we consider replacing "Here is a quick intro as to why adapters need to exist and how they are currently structured" with a sentence or two that sums up what adapters are at the top of the article?  I'm pretty new to the product, so this may not be entirely accurate, but something like "Adapters are the cornerstone of dbt's integrations with a variety of data warehouses. They are critical in translating information between dbt Core/Cloud and the database."

Reply:
@matthewshaver how's [this](https://github.com/dbt-labs/docs.getdbt.com/pull/1727/commits/2562f514d40783cb4e9981c5737364c1ac7a6782)?